usl_pipeline/usl_lib/usl_lib/readers/config_readers.py

```diff
@@ -21,3 +23,26 @@ def read_rainfall_amounts(rain_fd: TextIO) -> Sequence[float]:
             continue
         entries.append(float(rainfall_line.group(1)))
     return entries
+
+
+def read_key_value_pairs(fd: TextIO) -> dict:
+    """Reads a config file and returns a dictionary of k/v pairs.
+
+    Example format of file contents:
+    percentile = 99
+    simulation_year = 2012
+    simulation_months = JJA
+
+    Args:
+      fd: The file containing the key/value pairs
+
+    Returns:
+      A dictionary of the key/values pairs in the file
+    """
+    # Read lines from file first. If the first line is not blank and doesn't look
+    # like a section header, add [DEAULT] section so it is compliant with
+    # what ConfigParser expects
+    config = configparser.ConfigParser()
+    config.read_file(itertools.chain(["[DEFAULT]\n"], fd))
+    config_dict = {key: value for key, value in config["DEFAULT"].items()}
```

Comment:
It'd also be fine to `return config["DEFAULT"]` because `config["DEFAULT"]` is a dict-like (e.g. you can do `config["DEFAULT"]['simulation_year']`

can use the more generic Mapping
```python
read_key_value_pairs(fd: TextIO) -> Mapping
```
from the typing module to indicate a dict-like rather than specifically a dict

Reply:
but what you've got here is fine!